main.py

```diff
@@ -1,10 +1,37 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    # Load data from players.json
+    with open('players.json') as f:
+        players_data = json.load(f)
+
+    for player_data in players_data:
+        # Get or create the race
+        race, created = Race.objects.get_or_create(name=player_data['race'])
+
+        # Get or create the guild
+        guild, created = Guild.objects.get_or_create(name=player_data['guild'])
```

Comment:
The `get_or_create` method for `Guild` assumes that the 'guild' key is always present in `player_data`. If the JSON data might not include a guild for some players, consider handling the case where 'guild' is missing or is `None`. You could use a default value or handle the absence of a guild appropriately.